tests/functional/end-to-end.bats

```diff
@@ -374,3 +390,22 @@ test_one_check() {
 	echo ${output}
 	[ "$count" -eq 0 ]
 }
+
+@test "parse_error_printing" {
+	test_parse_error test1.te test1.output
+
+	test_parse_error test2.te test2.output
+
+	printf "#comment\n    policy_module( 	     test2, 	\n    \n\n	    	0.0.1  \n  )	\n\n" > ./policies/parse_errors/test3_tmp.if
```

Comment:
Because the test input contains trailing spaces/mixed tabs-spaces and windows line breaks.
They will cause problems with git and white-space checks if present in the repository.